src/eureka/S3_data_reduction/nirspec.py

```diff
@@ -280,15 +280,16 @@ def straighten_trace(data, meta, log):
     meta : eureka.lib.readECF.MetaClass
         The updated metadata object.
     '''
-    log.writelog('  Correcting curvature and bringing trace in the center '
-                 'of the detector...', mute=(not meta.verbose))
+    log.writelog('  Correcting G395H curvature and bringing the trace to the '
```

Comment:
NIRCam also has some curvature, so let's now specifically call out G395. Also, G235 may have curvature.

Reply:
Do you mean *not* instead of now?